docs/resources/samples/excel-samples.md

```diff
@@ -291,7 +291,7 @@ function main(workbook: ExcelScript.Workbook) {
 
 This sample finds the first PivotTable in the workbook and logs the values in the "Grand Total" cells (as highlighted in green in the image below).
 
-:::image type="content" source="../images/sample-pivottable-grand-total-row.png" alt-text="A PivotTable showing fruit sales with the Grand Total row highlighted green.":::
+:::image type="content" source="../../images/sample-pivottable-grand-total-row.png" alt-text="A PivotTable showing fruit sales with the Grand Total row highlighted green.":::
```

Comment:
```suggestion
:::image type="content" source="../../images/sample-pivottable-grand-total-row.png" alt-text="A PivotTable showing fruit sales with the Grand Total row highlighted green":::
```

Reply:
The alt-text periods were added on the accessibility advice of @lindalu-MSFT, though I'd love to double-check that I didn't misinterpret something.